app/src/main/java/org/coepi/android/ui/home/HomeViewModel.kt

```diff
@@ -4,55 +4,92 @@ import androidx.lifecycle.LiveData
 import org.coepi.android.ui.alerts.AlertsFragmentDirections.Companion.actionGlobalAlerts
 import org.coepi.android.ui.debug.DebugFragmentDirections.Companion.actionGlobalDebug
+import org.coepi.android.ui.home.HomeCardId.CHECK_IN
+import org.coepi.android.ui.home.HomeCardId.SEE_ALERTS
 import org.coepi.android.ui.navigation.NavigationCommand.ToDestination
 import org.coepi.android.ui.navigation.NavigationCommand.ToDirections
 import org.coepi.android.ui.navigation.RootNavigation
 import org.coepi.android.ui.symptoms.SymptomsFragmentDirections.Companion.actionGlobalSymptomsFragment
 
+enum class HomeCardId {
+    CHECK_IN, SEE_ALERTS
+}
+
 class HomeViewModel(
     private val rootNav: RootNavigation,
     envInfos: EnvInfos,
     private val resources: Resources,
     alertsRepo: AlertsRepo
 ) : ViewModel() {
 
-    val versionString: LiveData<String> =
-        just(resources.getString(home_version, envInfos.appVersionString()))
-            .toLiveData()
-
-    val newAlerts: LiveData<Boolean> = alertsRepo.alerts
+    private val newAlerts: LiveData<Boolean> = alertsRepo.alerts
         .map { it.isNotEmpty() }
         .startWith { false }
         .observeOn(mainThread())
         .toLiveData()
 
+    private val disposables = CompositeDisposable()
+
+    private val homeCardClickSubject: PublishSubject<HomeCard> = PublishSubject.create()
+
+    val versionString: LiveData<String> =
+        just(resources.getString(home_version, envInfos.appVersionString()))
+            .toLiveData()
+
     val title: LiveData<String> = alertsRepo.alerts
         .map { title(it.size) }
         .startWith(title(0))
         .observeOn(mainThread())
         .toLiveData()
 
+    val homeCardItems = listOf(
+        HomeCard(
+            CHECK_IN,
+            resources.getString(string.home_my_health_card_title),
+            resources.getString(string.home_my_health_card_description), newAlerts()
+        ),
+        HomeCard(
+            SEE_ALERTS,
+            resources.getString(string.home_contact_alerts_card_title),
+            resources.getString(string.home_contact_alerts_card_description), newAlerts()
+        )
+    )
 
-    fun onCheckInClick() {
-        rootNav.navigate(ToDestination(actionGlobalSymptomsFragment()))
+    init {
+        disposables += homeCardClickSubject
+            .subscribe { homeCardItem ->
+                when (homeCardItem.cardId) {
+                    CHECK_IN -> rootNav.navigate(ToDestination(actionGlobalSymptomsFragment()))
+                    SEE_ALERTS -> rootNav.navigate(ToDestination(actionGlobalAlerts()))
+                }
+            }
     }
 
-    fun onSeeAlertsClick() {
-        rootNav.navigate(ToDestination(actionGlobalAlerts()))
+    fun onClicked(item: HomeCard) {
+        homeCardClickSubject.onNext(item)
     }
 
     fun onDebugClick() {
         rootNav.navigate(ToDirections(actionGlobalDebug()))
     }
 
+    private fun newAlerts(): Boolean {
+        newAlerts.value?.let { return it }
```

Comment:
Retrieving the value of the subject like this makes the flow not reactive. If the alerts change in the background, the UI will not be updated. You have to make the cards (`homeCardItems`) depend on the `newAlerts` observable, such that they're generated again if it changes.

So something along the lines of converting `newAlerts` to observable (it's not observed directly by the UI anymore so it doesn't need to be `LiveData`), and applying `map` on it to generate `homeCardItems`.

Also, you probably should add `distinctUntilChanged` to `newAlerts`, such that if the background fetch doesn't find anything new, the UI is not updated.